### DISCUSSION TITLE
关于 Input 组件开发相关

### DISCUSSION THREAD

**Discussion by ghost** [FightingDesign/fighting-design#16] [Ideas]
## 基础配置项
- `type`  可选设置：`textarea` 控制input是否为文本域
- `size` 控制 大小
- `readonly` 是否为只读文本框
- `left-icon` 设置左侧 图标
- `right-icon` 设置右侧 图标
- `is-password` 是否是密码框
- `disabled` 是否禁用
- `max-length` 控制最大字数上限
- `min-length` 控制最小字数上限

## 深度配置
- `clearable` 是否在右侧显示一键清空文本框按钮

## 补充
另外更多配置可以根据实际的需求或者业务需要来随时增加

**Comment by Tyh2001** [comment]
最近我也计划一下，input 业务可能会比较多

**Comment by kiss5523** [comment]
边框圆角
边框颜色
获取焦点的颜色
失去焦点的颜色
如果是文本域：文本域边框的拖动大小，rows、cols 属性设置文本域的宽高

**Comment by Tyh2001** [comment]
我先出一下设计稿，莫急~~~

**Reply by ghost** [reply]
不急， 最近我在看面试题和 react🤔
